apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -88,29 +107,38 @@ def get(user, query_server=None, cluster=None):
         from impala.dbms import ImpalaDbms
         from impala.server import ImpalaServerClient
         DBMS_CACHE[user.id][query_server['server_name']] = ImpalaDbms(
-            HiveServerClientCompatible(ImpalaServerClient(query_server, user)),
-            QueryHistory.SERVER_TYPE[1][0]
+          HiveServerClientCompatible(ImpalaServerClient(query_server, user)),
+          QueryHistory.SERVER_TYPE[1][0]
         )
       elif query_server['server_name'] == 'hms':
         from beeswax.server.hive_metastore_server import HiveMetastoreClient
         DBMS_CACHE[user.id][query_server['server_name']] = HiveServer2Dbms(
-            HiveMetastoreClient(query_server, user),
-            QueryHistory.SERVER_TYPE[1][0]
+          HiveMetastoreClient(query_server, user),
+          QueryHistory.SERVER_TYPE[1][0]
         )
       else:
         from beeswax.server.hive_server2_lib import HiveServerClient
         DBMS_CACHE[user.id][query_server['server_name']] = HiveServer2Dbms(
-            HiveServerClientCompatible(HiveServerClient(query_server, user)),
-            QueryHistory.SERVER_TYPE[1][0]
+          HiveServerClientCompatible(HiveServerClient(query_server, user)),
+          QueryHistory.SERVER_TYPE[1][0]
         )
+    elif RESET_HS2_QUERY_SERVER:
+      from beeswax.server.hive_server2_lib import HiveServerClient, HiveServerClientCompatible
```

Comment:
Why imports?

Reply:
This was needed to reset set the dbms cache for a new hs2 due to the current HS2 going away, instead of clearing all of the DBMS cache.